yt_dlp/YoutubeDL.py

```diff
@@ -415,7 +415,13 @@ class YoutubeDL:
                        - Raise utils.DownloadCancelled(msg) to abort remaining
                          downloads when a video is rejected.
                        match_filter_func in utils.py is one example for this.
-    no_color:          Do not emit color codes in output.
+    color:             A Dictionary with output stream names as keys
+                       and their respective color policy as values.
+                       Can also just be a single color policy,
+                       in which case it applies to all outputs.
+                       Valid stream names are 'screen', 'stdout' and 'stderr'.
+                       Valid color policies are one of 'always', 'auto', 'nocolor' or 'never'.
+    no_color:          Same as `color='nocolor'`.
```

Comment:
The multiline printer already ignored the `--no-color` flag so the default for that flag should be `nocolor`. `--color never` disables sequences completely now.